lib/parser.dart

```diff
@@ -708,7 +708,7 @@ class _Parser {
 
         start = _peekToken.span;
         while (!_maybeEat(TokenKind.END_OF_FILE)) {
-          RuleSet ruleset = processRuleSet();
+          final ruleset = processRuleSet();
```

Comment:
Isn't the type inferred? I thought about renaming so it would be a surprise that `processRuleSet()` doesn't always return a `RuleSet` any longer. How about `processRules()` since it returns qualified rules and at-rules.